### DISCUSSION TITLE
Set a demand on a build pipeline to use only the upgraded version for an agent

### DISCUSSION THREAD

**Discussion by simonDominos** [actions/virtual-environments#5211] [General questions]
During the agent upgrade period, some agents in the pool (we use ubuntu 20.04) are still the older version and some are the newer upgraded verison. 

During this time, we have problems with some of our tests on the build pipeline as they require the version of chrome to be set - during this upgrade we see chrome change from v97 to v99 and our tests need a set version of chrome (can be either). Is there a way to set a demand in a pipeline to only pick a specific image version from the pool that has been upgraded? i.e. set a demand like: Agent.ImageVersion equals 20220306.1 ?
<img width="596" alt="Screenshot 2022-03-10 at 11 00 50" src="https://user-images.githubusercontent.com/48192735/157566213-4605e4d2-c72f-47c6-b914-235b3c7d5fb6.png">
<img width="913" alt="Screenshot 2022-03-10 at 11 14 07" src="https://user-images.githubusercontent.com/48192735/157567578-58cb56f9-5401-4541-9e04-b443d5c4ecf3.png">

**Comment by miketimofeev** [comment]
@simonDominos I'm afraid it is not possible at the moment. We are working on minimizing such gaps
